transforms/universal/ededup/python/src/ededup_transform_base.py

```diff
@@ -12,14 +12,23 @@
 
 from argparse import ArgumentParser, Namespace
 from typing import Any
+import pickle
 
 import pyarrow as pa
 from data_processing.transform import (
     AbstractTableTransform,
     AbstractTransform,
     TransformConfiguration,
 )
-from data_processing.utils import GB, CLIArgumentProvider, TransformUtils
+from data_processing.utils import (
+    GB,
+    CLIArgumentProvider,
+    TransformUtils,
+    UnrecoverableException,
+    str2bool,
+    get_logger
+)
+from data_processing.data_access import SnapshotUtils
 
 
 REQUEST_LEN = 8192
```

Comment:
Should the constants `short_name`  and `cli_prefix` be uppercase in the next lines?

Reply:
no rule for that

Reply:
there is a PEP8 recommendation to name constants with uppercase letters. 
https://peps.python.org/pep-0008/#constants

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.